README.md

```diff
@@ -24,34 +24,48 @@ More detailed requirements are outlined in the [Installing the Islandora Essenti
+### Global Fedora XACML policies
+Before installing Islandora, the XACML policies located [here](https://github.com/Islandora/islandora-xacml-policies) should be copied into the Fedora global XACML policies folder. This will allow "authenticated users" in Drupal to access Fedora API-M functions (create, edit, and delete objects in Fedora).
 
-Remove `deny-purge-datastream-if-active-or-inactive.xml` to allow for purging of datastream versions.
+Notes:
+* The `permit-upload-to-anonymous-user.xml` and `permit-apim-to-anonymous-user.xml` policies allow anonymous (unauthenticated)
+users to create objects and datastreams. They do not need to be present if this is not required.
+* The policy `deny-purge-datastream-if-active-or-inactive.xml` must be deleted to allow users to purge (permanently remove) datastream versions.
 
 More detailed information can be found in the 'Set XACML Policies' in the [Installing Fedora](https://wiki.duraspace.org/display/ISLANDORA/milestone+1+-+Installing+Fedora) chapter of the documentation.
 
+### Protecting the 'anonymous' username
+The `islandora_drupal_filter` passes the username of 'anonymous' through to Fedora for unauthenticated Drupal Users. A user with the name of 'anonymous' may have XACML policies applied to them that are meant to be applied to Drupal users that are not logged in or vice-versa. This is a potential security issue that can be plugged by creating a user named 'anonymous' and restricting access to the account. If this is done after installing Islandora, Drupal's cron can be run to remove expired authentication tokens.
+
+
 ## Configuration
 
-The `islandora_drupal_filter` passes the username of 'anonymous' through to Fedora for unauthenticated Drupal Users. A user with the name of 'anonymous' may have XACML policies applied to them that are meant to be applied to Drupal users that are not logged in or vice-versa. This is a potential security issue that can be plugged by creating a user named 'anonymous' and restricting access to the account.
+Configuration that applies to all solution packs, including the location of the Fedora Repository, the namespaces accessible by this instance of Islandora, and whether to generate derivatives on ingest, are available at  `admin/islandora/configure`.
+
+![Configuration](https://user-images.githubusercontent.com/1943338/40320855-724afcba-5d03-11e8-9109-0b8413349839.png)
 
-Drupal's cron can be run to remove expired authentication tokens.
+### Breadcrumb Generation
 
-**Breadcrumb Generation** on the configuration page, allows you to choose the default breadcrumb generation
- or a custom method (if implemented).
+Whether Drupal breadcrumbs (showing an object's parent hierarchy) should be displayed, and how they are generated, can be set on the configuration page. Other modules (such as [Islandora Solr](https://github.com/Islandora/islandora_solr_search)) may provide  alternatives that perform better at large scales than the built-in Resource Index query.
```

Comment:
"Islandora Solr Search", full module's name

Reply:
No, that's the repo name. Islandora Solr is what the module appears as in the module list.